docs/UserGuide.md

```diff
@@ -62,12 +59,12 @@ Additional info:
 
 * Basic Food Groups: `MEAT`, `SEAFOOD`, `EGG`, `DAIRY`, `VEGETABLE`, `FRUIT`,
   `BEVERAGE`, `COOKED_DISH`, `READY_TO_EAT`, `FROZEN`, `OTHERS`
-* Basic fridge location: `FREEZER`, `UPPER_SHELF`, `MIDDLE_SHELF`, `LOWER_SHELF`, 
+* Basic fridge location: `FREEZER`, `UPPER_SHELF`, `MIDDLE_SHELF`, `LOWER_SHELF`,
   `DRAWERS`, `FRIDGE_DOOR`, `OTHERS`
 
 Example of usage:
 
-```
+```lang-none
```

Comment:
What's lang-none?

Reply:
@Leeyp default notation requires a language to be defined for each code block, here declares that there is no language to highlight